deploy-board/deploy_board/templates/hosts/host_details.html

```diff
@@ -41,18 +41,19 @@
 {% endif %}
 
 {% if agent_wrappers %}
-    <div class="panel panel-default">
-    {% include "panel_heading.tmpl" with panel_title="Agent Details" panel_body_id="agentDetailsId" direction="down" copy_host_name_button=True %}
+<div class="panel panel-default">
+    {% include "panel_heading.tmpl" with panel_title="Agent Details" panel_body_id="agentDetailsId" direction="down"
+    copy_host_name_button=True %}
     <div id="agentDetailsId" class="collapse in panel-body">
-
-    {% for agent_wrapper in agent_wrappers %}
+        {% with hostdetailsvalue=host_details.items|lookup:"Phobos Link" %}
+        {% for agent_wrapper in agent_wrappers %}
         {% with env=agent_wrapper.env agent=agent_wrapper.agent %}
```

Comment:
can you move hostdetailsvalue=host_details.items|lookup:"Phobos Link" into here to have one less {% with ... %}

Reply:
updated, thanks for reminding.